docusaurus/docusaurus.config.js

```diff
@@ -11,7 +11,7 @@ const config = {
         'Airbyte is an open-source data integration platform to build ELT pipelines. Consolidate your data in your data warehouses, lakes and databases.',
     // this is non-functional, just a link back to OSS docs right now
     url: 'https://airbytehq.github.io',
-    baseUrl: '/airbyte/',
```

Comment:
we chatted about this before: is my understanding that this change is metadata only correct?

Reply:
I thought this was meta data only.  I need to update the comment.  I pushed from master without this change and it seems to effect docusaurus's idea of home.  Since we are running at `/` I removed the path here.  The above comment needs to be updated though.